Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -847,25 +850,40 @@ void Renderer::UpdateDrawRectangle()
   float draw_width = draw_aspect_ratio;
   float draw_height = 1;
 
-  // Crop the picture to a standard aspect ratio. (if enabled)
-  auto [crop_width, crop_height] = ApplyStandardAspectCrop(draw_width, draw_height);
-
-  // scale the picture to fit the rendering window
-  if (win_width / win_height >= crop_width / crop_height)
+  if (g_ActiveConfig.bIntegerScaling && m_last_xfb_width != 0 && m_last_xfb_height != 0)
```

Comment:
Is integer scaling *AND* crop a valid use case?  If not, we might want to do more so the user isn't confused by having both checkboxes checked.